NerdyNomads/Text-Savvy
# Delete a Workspace

## What does this PR do?

Add a delete option when editing a workspace, along with a popup confirming your choice.

### Screenshots (optional)

![image](https://user-images.githubusercontent.com/66437477/161406911-54411729-5a58-4da0-b811-615bca70f268.png)

![image](https://user-images.githubusercontent.com/66437477/161406914-5d305edc-6dbd-4fdd-b691-1a7fbd606a30.png)

### Steps to test/review (optional)

- Edit a workspace
- Click on the Delete button in the popup that appears
- Click outside the Confirmation Popup/Cancel button to go back to editing the workspace
- Click Delete again
- Click Confirm
- Check your console for a "Deleting Workspace with ID: " message.
- (Could add a reload here after deleting to refresh the workspace list)

## Checklist before merging
      - Reviewers
      - Assignee
      - Labels
      - Project
      - Milestone
      - Linked Issues
